acq4/devices/LEDLightSource/LEDLightSource.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from acq4.devices.Device import *
+from acq4.devices.LightSource import *
+from acq4.devices.DAQGeneric import DAQGeneric, DAQGenericTaskGui
+from PyQt4 import QtCore, QtGui
+import acq4.util.Mutex as Mutex
+
+class LEDLightSource(LightSource):
+    """Simple device which reports the status of the LED Light Sources...reports up to the LightSource object."""
+    
+    def __init__(self, dm, config, name):
+        LightSource.__init__(self, dm, config, name)
+        self.ledconfig = config.get('leds', config)
```

Comment:
I'm trying to get the leds that are defined in the devices.cfg.  The file looks like this:

_LEDs:
    driver: 'LEDLightSource'
    leds:
        blue:
            channel: 'LEDChannels', 'M490L4'
            model: 'Thorlabs.M490L4'
            wavelength: 490*nm
            power: 50*mW
        green:
            channel: 'LEDChannels', 'M530L4'
            model: 'Thorlabs M530L4'
            wavelength: 530*nm
        infrared:
            channel: 'LEDChannels', 'IR1'
            wavelength: 850*nm_

Should I removed the "leds" part, and just go right into the names?